core/src/test/java/org/infinispan/util/ControlledConsistentHashFactory.java

```diff
@@ -145,4 +152,51 @@ public void setOwnerIndexes(int[][] segmentOwners) {
          super.setOwnerIndexes(segmentOwners);
       }
    }
+
+   public static class Externalizer implements AdvancedExternalizer<ControlledConsistentHashFactory> {
```

Comment:
Shouldn't this be in the next PR?

Reply:
No we must add this Externalizer here, as the user marshaller needs to know how to marshall the `List<Address>` elements. Previously, this was not an issue as the GlobalMarshaller would utilise the configured `JGroupsAddress.Externalizer`. As the user marshaller is designed to only load the externalizers configured as part of the SerializationConfiguration, it's necessary to add a externalizer in the `NumOwnersNodeCrashInSequenceTest#createCacheManagers`. Looking at this again though, it's overkill to create an Externalizer for `ControlledConsistentHashFactory`, when it's sufficient to simply add the `JGroupsAddress.Externalizer`.